test_unstructured/file_utils/test_filetype.py

```diff
@@ -664,13 +696,13 @@ def and_it_returns_the_empty_string_as_the_extension_when_there_are_no_file_name
 
     # -- .file_head ---------------------------------------------
 
-    def it_grabs_the_first_4k_bytes_of_the_file_for_use_by_magic(self):
+    def it_grabs_the_first_8k_bytes_of_the_file_for_use_by_magic(self):
         ctx = _FileTypeDetectionContext(file_path=example_doc_path("norwich-city.txt"))
 
         head = ctx.file_head
 
         assert isinstance(head, bytes)
-        assert len(head) == 4096
+        assert len(head) == 8192
```

Comment:
This bump of `.file_head` size to `8192` has nothing specific to do with fixing the problem remedied by this PR, it was just too small to put in it's own PR so came along for the ride.